src/styles/loginSignupPage.scss

```diff
@@ -28,7 +30,7 @@
     margin-top: 2rem;
   }
 
-  .loginForm .targetSlogan {
+  .targetSlogan {
     color: #212121;
```

Comment:
Better to move to a constants file in SCSS and import them, you may want to reuse these